MaxText/llama_or_mistral_ckpt.py

```diff
@@ -957,7 +957,7 @@ def convert_to_jax_weights(base_model_path, model_size, huggingface_ckpt):
   return _convert_pytorch_to_jax_weights(base_model_path, model_size, model_params, mem_info)
 
 
-def save_weights_to_checkpoint(maxtext_model_path, jax_weights, device_count):
+def save_weights_to_checkpoint(maxtext_model_path, jax_weights, device_count, use_ocdbt=True, use_zarr3=True):
```

Comment:
is this intentional, it makes checkpoint with many files, right?

Perhaps we can pass those with command line arguments with default values?

Reply:
Updated.